java/gazelle/testdata/annotations/expectedStderr.txt

```diff
@@ -1 +0,0 @@
-{"level":"warn","_c":"maven-resolver","error":"open %WORKSPACEPATH%/maven_install.json: no such file or directory","message":"not loading maven dependencies"}
```

Comment:
Is it worth leaving one test that checks for this error message?

Reply:
It is, but for cross-platform reasons this is annoyingly fiddly. If it's ok with you, I'd like to merge this as-is, then add a feature to gazelle itself which would let us write this test reasonably, then add a new test which does this assertion?

Reply:
Ahh, so this is also part of the "Windows fixes" work right?